Source/Core/VideoCommon/VertexLoaderBase.cpp

```diff
@@ -229,7 +229,7 @@ std::unique_ptr<VertexLoaderBase> VertexLoaderBase::CreateVertexLoader(const TVt
 {
   std::unique_ptr<VertexLoaderBase> loader = nullptr;
 
-  // #define COMPARE_VERTEXLOADERS
+#define COMPARE_VERTEXLOADERS
```

Comment:
I assume you left this on by accident?

Reply:
Yep, I did. Though that does mean that we ran fifoci with `COMPARE_VERTEXLOADERS` enabled and it didn't detect any differences on x64 or ARM so that's a good sign.